Compilation failure due to incorrect return type in SPIRVTranspileContext

```
SDL_shadercross/src/SDL_shadercross.c:1028:24: error: incompatible types when returning type ‘_Bool’ but ‘SPIRVTranspileContext *’ was expected
 1028 |                 return false;
      |                        ^~~~~
SDL_shadercross/src/SDL_shadercross.c:1294:24: error: incompatible types when returning type ‘_Bool’ but ‘SPIRVTranspileContext *’ was expected
 1294 |                 return false;
      |                        ^~~~~
```

--- a/src/SDL_shadercross.c
+++ b/src/SDL_shadercross.c
@@ -1025,7 +1025,7 @@ static SPIRVTranspileContext *SDL_ShaderCross_INTERNAL_TranspileFromSPIRV(
             if (result < 0) {
                 SPVC_ERROR(spvc_resources_get_resource_list_for_type);
                 spvc_context_destroy(context);
-                return false;
+                return NULL;
             }
             num_texture_samplers = num_separate_samplers;
         }
@@ -1291,7 +1291,7 @@ static SPIRVTranspileContext *SDL_ShaderCross_INTERNAL_TranspileFromSPIRV(
             if (result < 0) {
                 SPVC_ERROR(spvc_resources_get_resource_list_for_type);
                 spvc_context_destroy(context);
-                return false;
+                return NULL;
             }
             num_texture_samplers = num_separate_samplers;
         }

